server/s3server.cc

```diff
@@ -812,12 +812,22 @@ int main(int argc, char **argv) {
 
   // Call this function at starting as we need to make use of our own
   // memory allocation/deallocation functions
-  rc = event_use_mempool(g_option_instance->get_libevent_pool_buffer_size(),
-                         g_option_instance->get_libevent_pool_initial_size(),
-                         g_option_instance->get_libevent_pool_expandable_size(),
-                         g_option_instance->get_libevent_pool_max_threshold(),
-                         mem_log_msg_func, libevent_mempool_flags);
-
```

Comment:
Try this instead of "if".  Will avoid copy-paste (and we get better stats with Codacy).
```
    rc = event_use_mempool(
        g_option_instance->get_libevent_pool_buffer_size(),
        g_option_instance->get_libevent_pool_initial_size(),
        g_option_instance->get_libevent_pool_expandable_size(),
        g_option_instance->get_libevent_pool_max_threshold(),
        g_option_instance->get_log_level() == "DEBUG" ? mem_log_msg_func : NULL,
        libevent_mempool_flags);
```

Reply:
ok